src/plugins/recordTypes/media/fields.js

```diff
@@ -0,0 +1,151 @@
+import { defineMessages } from 'react-intl';
+
+export default (configContext) => {
+  const {
+    AutocompleteInput,
+    CheckboxInput,
+    TextInput,
+    OptionPickerInput,
+  } = configContext.inputComponents;
+
+  const {
+    configKey: config,
+  } = configContext.configHelpers;
+
+  const {
+    DATA_TYPE_BOOL,
+  } = configContext.dataTypes;
+
+  return {
+    document: {
+      'ns2:media_common': {
+        identificationNumber: {
+          [config]: {
+            required: false,
+            view: {
+              type: TextInput,
+            },
+          },
+        },
+        title: {
+          [config]: {
+            defaultValue: 'media record',
+            view: {
```

Comment:
Don't need the view customization. In 4.1 we did a hack to set the default value, because the only way to set a default value was to add options. In 5.0 we can just set the defaultValue config property.